src/main/kotlin/com/exactpro/th2/lwdataprovider/entities/requests/SseEventSearchRequest.kt

```diff
@@ -118,8 +123,15 @@ data class SseEventSearchRequest(
             throw InvalidRequestException("One of the 'startTimestamp' or 'resumeFromId' must not be null")
     }
 
-    fun checkRequest() {
+    private fun checkRequest() {
         checkStartPoint()
         checkEndTimestamp()
     }
+
+    override fun toString(): String {
```

Comment:
Lets keep `startTimestamp` and `endTimestamp` closer to each other in `toString` method